### PR TITLE
Show real vCow and Cow data on profile page

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

- gets the real vCow and Cow data and shows it in the Profile page
- changes repo branch and vCow and Cow Rinkeby address

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @nenadV91 , great changes!
I can see actual claimed balance for different accounts. 
Some notes: 
1. Based on the discussion in the https://github.com/gnosis/cowswap/pull/2564#issuecomment-1074185848v, we should show card with 0 COW balance + banner when a user is not connected
2. The same is when a user has 0 COW in the wallet

However, I'm not sure if this is related to the current task..

**Comment by nenadV91** [comment]
@anxolin Updated this was created before @fairlighteth merged his PR and that is why it was just a draft, but its fixed now.

File: src/custom/pages/Profile/index.tsx [hunk 1]
```diff
@@ -55,27 +52,25 @@ export default function Profile() {
...
+  const vCowBalance = formatSmartLocaleAware(total, AMOUNT_PRECISION) || '0'
 
-  // TODO: remove once this is not needed anymore
-  // if (isPr || isLocal) {
-  //   console.force.log('vested', formatSmartLocaleAware(vested, vested?.currency.decimals))
-  //   console.force.log('total', formatSmartLocaleAware(total, total?.currency.decimals))
-  //   console.force.log('unvested', formatSmartLocaleAware(unvested, unvested?.currency.decimals))
-  //   console.force.log('cowBalance', formatSmartLocaleAware(cowBalance, cowBalance?.currency.decimals))
-  // }
+  const hasCowBalance = cow && !cow.equalTo(0)
```
Comment (W3stside) [inline src/custom/pages/Profile/index.tsx:66]:
```suggestion
  const hasCowBalance = !cow?.equalTo(0)
```

File: src/custom/pages/Profile/index.tsx [hunk 2]
```diff
@@ -55,27 +52,25 @@ export default function Profile() {
...
+  const hasVCowBalance = total && !total.equalTo(0)
```
Comment (W3stside) [inline src/custom/pages/Profile/index.tsx:67]:
```suggestion
  const hasVCowBalance = !total?.equalTo(0)
```

File: src/custom/state/claim/hooks/index.ts [hunk 3]
```diff
@@ -62,7 +62,9 @@ import useIsMounted from 'hooks/useIsMounted'
 import { ChainId } from '@uniswap/sdk'
 import { ClaimInfo } from 'state/claim/reducer'
 
-const CLAIMS_REPO_BRANCH = 'gip-13'
+// TODO: Revert this before it goes to production
+// const CLAIMS_REPO_BRANCH = 'gip-13'
```
Comment (W3stside) [inline src/custom/state/claim/hooks/index.ts:66]: revert?

**Comment by W3stside** [comment]
please point to #2573

**Comment by nenadV91** [comment]
@W3stside Is the latest @fairlighteth PR with the swap UI cards merged into release branch?

File: src/custom/pages/Profile/index.tsx (hunk 2 above)
Comment (nenadV91) [reply src/custom/pages/Profile/index.tsx]: updated

File: src/custom/state/claim/hooks/index.ts (hunk 3 above)
Comment (nenadV91) [reply src/custom/state/claim/hooks/index.ts:66]: Change back, that is what I meant

File: src/custom/pages/Profile/index.tsx (hunk 1 above)
Comment (nenadV91) [reply src/custom/pages/Profile/index.tsx]: Actually this doesn't work as expected https://jsfiddle.net/pksz7fy6/

File: src/custom/pages/Profile/index.tsx (hunk 2 above)
Comment (nenadV91) [reply src/custom/pages/Profile/index.tsx]: Same as previous case

File: src/custom/pages/Profile/index.tsx (hunk 1 above)
Comment (W3stside) [reply src/custom/pages/Profile/index.tsx:66]: Ah ok, I was curious about the negative bool check and undefined

**Comment by W3stside** [comment]
Why ar you pointing back at develop?

File: src/custom/state/claim/hooks/index.ts (hunk 3 above)
Comment (W3stside) [reply src/custom/state/claim/hooks/index.ts:66]: No I mean like, change it back. In this pr

**Comment by nenadV91** [comment]
Because its easier to review and if I change to release, changed files will also include the ones from this PR #2564 since that PR is not in the release branch.

**Review by anxolin — APPROVED** [review]
Nice!

**Pull request merged**